src/System.Net.HttpListener/src/System/Net/Windows/WebSockets/HttpWebSocket.Windows.cs

```diff
@@ -18,20 +18,17 @@ internal static partial class HttpWebSocket
         private static readonly Random s_keyGenerator = new Random();
         private static readonly bool s_httpSysSupportsWebSockets = (Environment.OSVersion.Version >= new Version(6, 2));
 
-        internal static ArraySegment<byte> EmptyPayload
-        {
-            get { return s_EmptyPayload; }
```

Comment:
Nit: `new ArraySegment<byte>(new byte[] { }, 0, 0)` above can be `new ArraySegment<byte>(Array.Empty<byte>())`